L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker_cfg.py

```diff
@@ -42,27 +42,28 @@
 # input and output
 ############################################################
 
-process.maxEvents = cms.untracked.PSet(input = cms.untracked.int32(20))
+
```

Comment:
All the configurations in this file should be set back to the default configurations used by everyone. So the maxEvents, readFiles, skipEvents, useJobReport, fileName, etc.

Reply:
Reverted these changes as part of [d95ffe9](https://github.com/cms-l1t-offline/cmssw/pull/1257/commits/d95ffe97501c7e003b3bc9de7e83685ecc20a6a6)